developer_guides/debugability/logger/index.rst

```diff
@@ -128,6 +128,15 @@ Examples
      sof-logger -l ldc_file -d
 
 
+.. note::
+  debugfs files used by ``sof-logger``
+
+  - ``etrace``: direct access (memcpy) to TRACE window of the SOF firmware
+  - ``trace``: Using DMA to stream debug trace information from SOF firmware (on
```

Comment:
either both capital or both lowercase "direct" and "using"

Reply:
OK, I'll convert to lowercase.